src/dialects/postgres/query-generator.js

```diff
@@ -863,18 +863,6 @@ export class PostgresQueryGenerator extends PostgresQueryGeneratorTypeScript {
     return dataType;
   }
 
-  /**
-   * Generates an SQL query that returns all foreign keys of a table.
-   *
-   * @param  {string} tableName  The name of the table.
-   * @returns {string}            The generated sql query.
-   * @private
-   */
-  getForeignKeysQuery(tableName) {
-    return 'SELECT conname as constraint_name, pg_catalog.pg_get_constraintdef(r.oid, true) as condef FROM pg_catalog.pg_constraint r '
-      + `WHERE r.conrelid = (SELECT oid FROM pg_class WHERE relname = '${tableName}' LIMIT 1) AND r.contype = 'f' ORDER BY 1;`;
```

Comment:
Not sure why, but this trailing semicolon made it so that more was being queried. See the following output from one of the tests;
```
[
  {
    constraint_name: 'hosts_admin_fkey',
    condef: 'FOREIGN KEY (admin) REFERENCES users(id)',
    id: 'hosts_admin_fkey',
    table: 'users',
    from: 'admin',
    to: 'id'
  },
  {
    constraint_name: 'hosts_operator_fkey',
    condef: 'FOREIGN KEY (operator) REFERENCES users(id) ON UPDATE CASCADE',
    id: 'hosts_operator_fkey',
    table: 'users',
    from: 'operator',
    to: 'id',
    on_update: 'CASCADE'
  },
  {
    constraint_name: 'hosts_owner_fkey',
    condef: 'FOREIGN KEY (owner) REFERENCES users(id) ON UPDATE CASCADE ON DELETE SET NULL',
    id: 'hosts_owner_fkey',
    table: 'users',
    from: 'owner',
    to: 'id',
    on_update: 'CASCADE'
  }
]
```
Since that test did a raw query with this function, this behaviour seems incorrect (we only ask for constraint_name and condef) and is fixed